composer.json

```diff
@@ -90,7 +90,8 @@
         "classmap": [
             "stubs",
             "rules-tests/CodingStyle/Rector/Namespace_/ImportFullyQualifiedNamesRector/Source",
-            "rules-tests/Renaming/Rector/Name/RenameClassRector/Source"
+            "rules-tests/Renaming/Rector/Name/RenameClassRector/Source",
+            "rules-tests/TypeDeclaration/Rector/ClassMethod/KnownMagicClassMethodTypeRector/Source"
```

Comment:
This should not be needed. The class respects PSR-4, right?

Reply:
It is weird that without it, it doesn't work on this case, probably some sort order issue or some case sensitive typo, but double check without add to classmap, use previous return null, then run "composer update", and error happen

Reply:
I created new PR for it:

- https://github.com/rectorphp/rector-src/pull/7427